content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
+Each `Rule` is intended to represent one *business rule* that should be implemented.
+A `Rule` is used to group together several `Example` (aka: `Scenario`) / `Scenario Outline`
+that belong to this *business rule*. In addition, a `Rule` may contain a `Background` section.
+
+Rule parent:
+
+- `Feature` (cardinality: one)
+
+Rule children:
+
+- `Description` (cardinality: optional, `0 .. 1`)
+- `Background` (cardinality: optional, `0 .. 1`)
+- `Example`/`Scenario` (cardinality: many, `0 .. N`)
+- `Scenario Outline`/`Scenario Template` (cardinality: many, `0 .. N`)
+
+Rule constraints:
+
+- A `Rule` should have at least one child (that is neither `Description` nor `Background`).
```

Comment:
Is this checked by the parser? What if I have a `Rule` but no scenario/example?

Reply:
Therefore, I use the word "should".  A `Rule` w/o `Example` makes no sense (IMHO) but is valid Gherkin.

Currently, I am not running the Gherkin parser. Therefore, I do not know.
A while ago, there was a discussion about a `Gherkin linter`. But I am not quite sure how it ended. The `linter` should provide such warnings (if the Gherkin parser does not).